.eslintrc.json

```diff
@@ -93,7 +93,7 @@
         "functions": "never"
       }
     ],
-    "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/*.test.js", "**/*.test.ts"] }],
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
```

Comment:
Couldn't work out why it only complained about the type for uuid and no other modules?

Reply:
not sure this is right? Doesn't the change says that it shouldn't complain if you use a dev import in any code including prod? 

I think you just need to install `uuid` explicitly as a prod dependency?